.travis.yml

```diff
@@ -3,7 +3,10 @@ os: linux
 
 arch:
   - amd64
-  - arm64
+  # It takes a very long time to spin up an ARM machine on travis.
+  # Because of this, it takes a long time before we can merge PRs.
+  # So this is disabled for now. We can re-enable if we need this.
+  #- arm64
```

Comment:
Someone could mark this arm64 as [allow_failures](https://docs.travis-ci.com/user/build-matrix/#rows-that-are-allowed-to-fail) and setup travis to [fast-finishing](https://docs.travis-ci.com/user/build-matrix/#fast-finishing). In that way, the build will report success without waiting for this arm64 build to finish.

Reply:
cc @jmazzitelli ^

Reply:
that sounds like a fine PR someone in the community can contribute if they want this feature.